### PR TITLE
fix regex when checking containerd config.toml

### PR DESCRIPTION
**Pull request by wyhysj**
## Description

There is incorrect regex in dfdaemon-daemonset, when injecting registry auth into containerd config.toml, will generate duplicate entries, which leads to containerd down after changing config.toml

Fix the regex, then should be okay.

### CODE REVIEW

**Comment by gaius-qi** [comment]
@wyhysj 

You should update `Chart.yaml` version:
1. https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L6
2. https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L7

Add changelog to `Chart.yaml`:
https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L28

**Comment by wyhysj** [comment]
> @wyhysj
> 
> You should update `Chart.yaml` version:
> 
> 1. https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L6
> 2. https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L7
> 
> Add changelog to `Chart.yaml`: https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/Chart.yaml#L28

@gaius-qi bumped chart version and added changelog.

File: charts/dragonfly/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: dragonfly
 description: Dragonfly is an intelligent P2P based image and file distribution system
 icon: https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/docs/images/logo/dragonfly.svg
 type: application
-version: 1.0.3
-appVersion: 1.0.3
+version: 1.0.4
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:6]: Update version to `1.0.5`.

File: charts/dragonfly/Chart.yaml [hunk 2]
```diff
@@ -3,8 +3,8 @@ name: dragonfly
...
+appVersion: 1.0.4
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:7]: Update version to `1.0.5`.

File: charts/dragonfly/Chart.yaml [hunk 3]
```diff
@@ -26,6 +26,7 @@ sources:
 
 annotations:
   artifacthub.io/changes: |
+    - Update regex when checking containerd config.toml.
     - Update image version to v2.1.0-alpha.6.
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:30]: Remove previous changelog。

File: charts/dragonfly/Chart.yaml [hunk 4]
```diff
@@ -26,6 +26,7 @@ sources:
...
     - Add redis configuration in scheduler.
```
Comment (gaius-qi) [inline charts/dragonfly/Chart.yaml:31]: Remove previous changelog。

**Review by gaius-qi — CHANGES_REQUESTED** [review]

**Comment by gaius-qi** [comment]
@wyhysj Please fix request changes. Thanks！

**Comment by wyhysj** [comment]
> @wyhysj Please fix request changes. Thanks！

@gaius-qi bumped version and removed previous changelog.

**Pull request merged**